lighthouse-core/audits/installable-manifest.js

```diff
@@ -51,59 +143,112 @@ class InstallableManifest extends MultiCheckAudit {
   }
 
   /**
-   * @param {LH.Artifacts.ManifestValues} manifestValues
-   * @return {Array<string>}
+   * @param {LH.Artifacts} artifacts
+   * @return {{i18nErrors: Array<LH.IcuMessage | string>; warnings: Array<LH.IcuMessage | string>}}
    */
-  static assessManifest(manifestValues) {
-    if (manifestValues.isParseFailure && manifestValues.parseFailureReason) {
-      return [manifestValues.parseFailureReason];
+  static getInstallabilityErrors(artifacts) {
+    const installabilityErrors = artifacts.InstallabilityErrors.errors;
+    const i18nErrors = [];
+    const warnings = [];
+    const errorArgumentsRegex = /{([^}]+)}/g;
+
+    for (const err of installabilityErrors) {
+      let matchingString;
+
+      // Filter out errorId 'in-incognito' since Lighthouse recommends incognito.
+      if (err.errorId === 'in-incognito') continue;
+
+      if (err.errorId === 'warn-not-offline-capable') {
```

Comment:
lets add a test to capture this warning case? ;)

Reply:
is this easy to add to the non-offline smoke tests sites, too?